### DISCUSSION TITLE
Issue with my AFSC lab manual showing up for everyone but me

### DISCUSSION THREAD

**Discussion by cberchok** [nmfs-ost/PAM-OpenScience#9] [General]
I clicked on the link through my AFSC organization page and now I'm seeing the changes.

**Comment by Kourtney-Burger** [comment]
Hi Catherine,

That's great that you can see those changes now! Such a weird bug. 

As a reminder you can tag people using the in these discussions using the @ symbol if you want certain people to see it (like this @cberchok).

**Reply by cberchok** [reply]
I spoke with Josh London and he said that he always makes a point to log into the AFSC organization before he does anything with his repo, so that might help others if they run into the same issue.  @Kourtney-Burger

**Reply by Kourtney-Burger** [reply]
Thanks for letting us know! Yeah, it seems like sometimes the sign on times out so that's definitely good practice to make sure you are logged into your org before you do anything else
